### PR TITLE
[NO GBP] More balloon alerts for guns + fixes a wrong alert

### PR DESCRIPTION
**Pull request by msgerbs**
## About The Pull Request

Turns out I missed a few to_chats in #70398. Also I accidentally inverted one so it said close the cover instead of open it. Oops!
## Why It's Good For The Game

Same reason as the original PR, balloon alerts are handy for stuff you care about right now and not in 5 minutes. Also they should probably not tell you to do the opposite of what you actually need to do.
## Changelog
:cl: VexingRaven
fix: Fixed a few balloon alerts for guns and moved a few more chat messages to balloon alerts
/:cl:

### CODE REVIEW

File: code/datums/components/fullauto.dm [hunk 1]
```diff
@@ -279,7 +279,7 @@
 		return FALSE
 	var/obj/item/bodypart/other_hand = shooter.has_hand_for_held_index(shooter.get_inactive_hand_index())
 	if(weapon_weight == WEAPON_HEAVY && (shooter.get_inactive_held_item() || !other_hand))
-		to_chat(shooter, span_warning("You need two hands to fire [src]!"))
+		balloon_alert(shooter, "need two hands!")
```
Comment (ShizCalev) [inline code/datums/components/fullauto.dm:282]: needs perhaps?

Comment (msgerbs) [reply code/datums/components/fullauto.dm:282]: Actually after thinking a bit I think `use both hands!` is better wording, what do you think?

Comment (msgerbs) [reply code/datums/components/fullauto.dm:282]: Otherwise my reasoning for `need two hands` is that it is short for you "You need two hands"

Comment (ShizCalev) [reply code/datums/components/fullauto.dm:282]: > Actually after thinking a bit I think `use both hands!` is better wording, what do you think?

i dig it

**Pull request merged**